src/tools/perf/perftest.c

```diff
@@ -39,7 +39,7 @@ test_type_t tests[] = {
      "atomic add latency", "latency", 1},
 
     {"get", UCX_PERF_API_UCT, UCX_PERF_CMD_GET, UCX_PERF_TEST_TYPE_STREAM_UNI,
-     "get latency / bandwidth / message rate", "latency", 1},
+     "get latency / bandwidth / message rate", "latency", 32},
```

Comment:
we should create 2 new tests here: get and get_bw.
since "get" will not test "latency" anymore, when using larger window.
and add the new test to CI